src/plugins/virtual_scroll/plugin.ts

```diff
@@ -207,8 +207,19 @@ export default function (this: TomSelect) {
 				return;
 			}
 
+			// Save scroll position.
+			const scrollTop = dropdown_content.scrollTop;
+
 			loading_more = true;
 			self.load.call(self, self.lastValue);
+
+			// Restore scroll position after loading more results.
+			const observer = new MutationObserver(() => {
+				dropdown_content.scrollTop = scrollTop + 10;
```

Comment:
I know this is arbitary, but I am not sure what I am missing here because `scrollTop` seemed to be out of sync with the exact position. Any thoughts @veryspry ?

Reply:
Yeah, I tested this and see what you mean